src/app/unit-host/unit-host.component.ts

```diff
@@ -356,23 +356,22 @@ export class UnitHostComponent implements OnInit, OnDestroy {
             }
             if (msgData.unitState) {
               const unitState = msgData.unitState;
-              const presentationProgress = unitState['presentationProgress'];
+              const { presentationProgress } = unitState;
```

Comment:
Also das hier hättest du garnicht ändern müssen. Ich glaube wir hatten uns im Team irgendwann mal darauf geeinigt, dass wir beide zulassen, aber die alte Variante sogar besser finden und die entsprechende ESLint-Regel angepasst.

Reply:
Dann stimmt offensichtlich noch immer nicht alles mit meinen Linter. Solche Zugriffe mit eckige Klammern und Strings, werden bei mir als Fehler angezeigt

Reply:
Ah das liegt vermutlich daran, dass da eine alte Version drin ist, in der noch nicht alle unsere Anpassungen sind. Update den gerne mal auf die 12er Version.
https://www.npmjs.com/package/@iqb/eslint-config